src/main/java/com/knighttodo/knighttodo/controller/IsReadyController.java

```diff
@@ -0,0 +1,33 @@
+package com.knighttodo.knighttodo.controller;
+
+import com.knighttodo.knighttodo.entity.Todo;
+import com.knighttodo.knighttodo.service.TodoService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+
+@RestController
+@RequestMapping("/ready")
+public class IsReadyController {
+
+    private TodoService todoService;
```

Comment:
Here it's better to use final, it's a best practice, I'll tell you about it leter.

Reply:
Okay